src/routes/_components/status/StatusHeader.html

```diff
@@ -1,6 +1,12 @@
 <div class="status-header {isStatusInNotification ? 'status-in-notification' : ''} {notificationType === 'follow' ? 'header-is-follow' : ''}">
   <div class="status-header-avatar {timelineType === 'pinned' || notificationType === 'poll' ? 'hidden' : ''}">
-    <Avatar {account} size="extra-small"/>
+    <a id={elementId}
```

Comment:
So right now this link is duplicating the text based 'Person has boosted' link. This is good for pointer users e.g. touch or mouse but for screen reader users it's duplicate.

Using 'title' to give the link a name is not sufficient for screen readers.

My recommendation here is to intentionally remove this duplicate link from being focusable with a keyboard or discoverable by a screen reader.

```html
<a id="..." href="..." aria-hidden="true" tabindex="-1">
   <img class="..." aria-hidden="true" alt="" title="" src="...">
</a>
```